http-common/Cargo.toml

```diff
@@ -10,17 +10,21 @@ base64 = "0.13"
 futures-util = "0.3"
 http = "0.2"
 hyper = { version = "0.14", features = ["client", "http1", "server", "tcp"], optional = true }
+hyper-openssl = { version = "0.9", optional = true }
+hyper-proxy = { git = "https://github.com/massand/hyper-proxy.git", branch = "sslstream", optional = true }
```

Comment:
Will there be a follow-up PR that switches this over to https://github.com/Azure/hyper-proxy, or is this PR blocked until the fork PR is approved?

Reply:
We can wait for the fork update. We rather use the final repo in our dependencies. If you have other comments, let me know.

Reply:
Updated.